components/binary_sensor/lvgl.rst

```diff
@@ -0,0 +1,43 @@
+.. _lvgl-bse:
+
+LVGL Binary Sensor
+==================
+
+.. seo::
+    :description: Instructions for setting up a LVGL widget binary sensor.
+    :image: ../images/logo_lvgl.png
+
+The ``lvgl`` binary sensor platform creates a binary sensor from a LVGL widget
+and requires :ref:`LVGL <lvgl-main>` to be configured.
+
+Supported widgets are ``btn`` and ``checkbox``. A single binary sensor supports
+a single widget, thus you need to choose among which one's state you want to use.
+
+
+Configuration variables:
+------------------------
+
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+- **name** (**Required**, string): The name of the binary sensor.
+- **widget** (**Required**): The ID of a button widget configured in LVGL.
+- All other options from :ref:`Binary Sensor <config-binary_sensor>`.
+
+
+Example:
+
+.. code-block:: yaml
+
```

Comment:
is it an idea to add the **lvgl** widget's that can be used with the documented sensor here as well?

Reply:
As a link should be enough.

Reply:
Yes that could do as well indeed.

Reply:
https://github.com/esphome/esphome-docs/pull/3510/commits/543d8f4ee5529d392bd79ba439f150c5deb664c0